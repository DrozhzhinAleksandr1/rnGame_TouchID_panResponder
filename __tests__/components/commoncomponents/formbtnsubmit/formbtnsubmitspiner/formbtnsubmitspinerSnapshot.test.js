import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import ShowSpiner from '../../../../../app/components/commoncomponents/formbtnsubmit/formbtnsubmitspiner/formbtnsubmitspiner';

it('create snapshot ShowSpiner', () => {
    const renderer = new ShallowRenderer()
    let toolbarSnapshot = renderer.render(<ShowSpiner />);
    expect(toolbarSnapshot).toMatchSnapshot();
});