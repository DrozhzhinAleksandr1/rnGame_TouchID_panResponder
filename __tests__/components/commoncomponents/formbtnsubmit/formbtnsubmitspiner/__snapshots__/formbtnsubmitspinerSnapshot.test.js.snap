// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`create snapshot ShowSpiner 1`] = `null`;
